lambda/custom/__tests__/integration/handlers.test.js

```diff
@@ -71,6 +77,7 @@ describe('Handlers testing', () => {
       mockTopHeadLines.mockImplementation(() => Promise.resolve(NEWS_API_SUCCESS_1_ARTICLE_RES));
 
       mockContext.event = ALEXA_NEWS_INQUIRY_REQUEST;
+      mockContext.attributes = {};
```

Comment:
should this also be set for the next test (line 95)?

Reply:
No it is not necessary on L95 because on the test on L95 attributes [this line](https://github.com/gtonye/voice-tech-to-3/pull/1/files/6163f8e78970ed2ef36dfb7ef0ca8c3cd68d97fa#diff-4a7ef796f11f6075f419a9bdfd3df9d5R85) and [this line](https://github.com/gtonye/voice-tech-to-3/pull/1/files/6163f8e78970ed2ef36dfb7ef0ca8c3cd68d97fa#diff-4a7ef796f11f6075f419a9bdfd3df9d5R86) for which the mock attributes property initialization is necessary are never called in the context of that test (the code errors out before).